### PR TITLE
Add HMX on Blast

### PR DESCRIPTION
**Pull request by shutsuwei**
**NOTE**

#### Please enable "Allow edits by maintainers" while putting up the PR.

---

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
3. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
4. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
5. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
6. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/perp88 exports TVL: 
        
 
 
    blast                     46.05 k
    polygon                   11.65 k
    arbitrum                  28.24 M
    
    total                    28.30 M

File: projects/perp88/index.js [hunk 1]
```diff
@@ -10,6 +11,48 @@ const tokens = {
 }
 const POOL_DIAMOND_CONTRACT = '0xE7D96684A56e60ffBAAe0fC0683879da48daB383';
 
+async function blastTvl(_, _1, _2, { api }) {
+  const vaultStorageAddress = '0x97e94BdA44a2Df784Ab6535aaE2D62EFC6D2e303'
+  const hlpUnderlyings = [
```
Comment (g1nt0ki) [inline projects/perp88/index.js:16]: added code to price the yield bearing tokens here: https://github.com/DefiLlama/defillama-server/commit/ea66a702a064c0cd8c39aa2120fcde8686831017#diff-31b49e7a59545ee10ded3ab0cfc3429ad5a5365b3495a7a2c0fbb0eb804d3fa8R39 so this adaper can be simplified

**Comment by g1nt0ki** [comment]
@shutsuwei thanks for the PR

**Comment by llamatester** [comment]
The adapter at projects/perp88 exports TVL: 
        
 
 
    polygon                   11.66 k
    blast                     0
    arbitrum                  28.27 M
    
    total                    28.28 M

**Pull request merged**